packages/syft/src/syft/service/user/user_service.py

```diff
@@ -325,11 +328,43 @@ def admin_verify_key(self) -> Union[SyftVerifyKey, SyftError]:
         except Exception as e:
             return SyftError(message=str(e))
 
+    @service_method(path="user.toggle_registration", name="toggle_signup")
```

Comment:
can we rename this? when I read the word toggle as a user I would expect changing from false->true when false and true->false when true. But instead it seems like we are setting the value